astroid/node_classes.py

```diff
@@ -2837,6 +2837,21 @@ def catch(self, exceptions): # pylint: disable=redefined-outer-name
         return False
 
 
+class FrozenSet(_BaseContainer):
```

Comment:
Hey Bryce, this was actually in objects because there is no frozenset AST node in Python, if I recall correctly. Can you check and revert this change if so?

Reply:
I mean, there's no Instance node in the AST as well

Reply:
Hm but `Instance` is in `Bases`

Reply:
@PCManticore I could put `FrozenSet` into its own file?

Reply:
Yeah I'll revert it

Reply:
Just wanted to reply to the deleted comments: yes, Instance is in `bases`, but that's because it's needed as a base class for some of the nodes. It's also a bit more generic than what we have in `objects`, since those classes are more specialised versions of an instance of *something*.
Regarding the circular import, in the 2.0 branch that we started some years ago with @ceridwen  there are some changes to alleviate a part of these circular dependencies, although not all of them were addressed. Once I'll finish what I have in store for 3.7 support, I'll focus on bringing those changes to master.